test/functional/wallet_taproot.py

```diff
@@ -387,6 +390,14 @@ def do_test_psbt(self, comment, pattern, privmap, treefn, keys_pay, keys_change)
         psbt_online.unloadwallet()
         psbt_offline.unloadwallet()
 
+    def do_test_k_of_n(self, k, n):
+        self.do_test(
+            f"tr(XPUB,multi_a({k},XPRV_1,...,XPRV_N)",
+            f"tr($2/*,multi_a({k}" + (",$1/*" * n) + "))",
+            [True, False],
```

Comment:
Note to other reviewers: this `privmap` argument determines for which keys an xpriv is inserted into the descriptor.